### PR TITLE
Support latest dart_style

### PR DESCRIPTION
**Pull request by simolus3**

### CODE REVIEW

File: build_runner/pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: build_runner
-version: 1.12.1
+version: 1.12.2-dev
```
Comment (jakemac53) [inline build_runner/pubspec.yaml:2]: lets go ahead and get this out the door, I know a few people are waiting on it
```suggestion
version: 1.12.2
```

File: build_runner/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 1.12.2-dev
```
Comment (jakemac53) [inline build_runner/CHANGELOG.md:1]:
```suggestion
## 1.12.2
```

**Review by jakemac53 — APPROVED** [review]

**Comment by jakemac53** [comment]
Oh actually it looks like this did land as a part of https://github.com/dart-lang/build/pull/3053/files#diff-73abdb8368e1118f5e506f67f4d0b15076a37085338b8f4b4803ecc5b434b91a

**Comment by kevmoo** [comment]
@simolus3 – THIS IS NOT PERSONAL! I'm so sorry!

Ask @jakemac53 and @natebosch about me when dealing without outdated dependencies – I get a bit 🥜

**Comment by simolus3** [comment]
No worries, it makes sense to bump all the dependencies in one PR and yours was there first :)

**Pull request closed without merging**